pyigm/surveys/igmsurvey.py

```diff
@@ -300,13 +340,97 @@ def update_mask(self, mask, increment=False):
         else:
             raise ValueError('abs_survey: Needs developing!')
 
+    def __getattr__(self, k):
+        """ Generate an array of attribute 'k' from the IGMSystems
+
+        Mask is applied
+
+        Parameters
+        ----------
+        k : str
+          Attribute
+
+        Returns
+        -------
+        numpy array
+        """
+        try:
+            lst = [getattr(abs_sys, k) for abs_sys in self._abs_sys]
+        except ValueError:
+            raise ValueError("Attribute does not exist")
+        # Special cases
+        if k == 'coord':
+            ra = [coord.ra for coord in lst]
+            dec = [coord.dec for coord in lst]
+            lst = SkyCoord(ra=ra, dec=dec)
+            return lst[self.mask]
+        # Recast as an array
+        return lst_to_array(lst, mask=self.mask)
+
+    def __add__(self, other, toler=2*u.arcsec):
+        """ Combine one or more IGMSurvey objects
+
+        Routine does a number of checks on the abstype,
+        the uniqueness of the sightlines and systems, etc.
+
+        Parameters
+        ----------
+        other : IGMSurvey
+        toler : Angle or Quantity
+          Tolerance for uniqueness
+
+        Returns
+        -------
+        combined : IGMSurvey
+
+        """
+        # Check the Surveys are the same type
+        if self.abs_type != other.abs_type:
```

Comment:
But this was only true for the IGMSurvey classes.

I think it was good to expunge the other print statements.